packages/create-sitecore-jss/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "npm run clean && tsc && ts-node ./scripts/build-templates.ts",
     "clean": "del-cli dist types",
     "lint": "eslint --no-eslintrc -c .eslintrc ./src/**/*.ts",
-    "watch": "ts-node ./scripts/watch-templates.ts",
+    "watch": "ts-node ./scripts/watch-templates.ts && ./scripts/post-install.js",
```

Comment:
`watch-templates` doesn't exit from the process, as I remember, did you test and check that `post-install` will be executed?

Reply:
@illiakovalenko yes I have tested the changes, post-install gets executed and the watch script doesn't exit.